dietpi/dietpi-software

```diff
@@ -11929,40 +11931,45 @@ _EOF_
+			if [[ ! -f '/mnt/dietpi_userdata/vaultwarden/vaultwarden.env' ]]
 			then
-				G_EXEC cp /opt/bitwarden_rs/.env.template /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
-				G_CONFIG_INJECT 'DATA_FOLDER=' 'DATA_FOLDER=/mnt/dietpi_userdata/bitwarden_rs' /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
+				G_EXEC cp /opt/vaultwarden/.env.template /mnt/dietpi_userdata/vaultwarden/vaultwarden_rs.env
+				G_CONFIG_INJECT 'DATA_FOLDER=' 'DATA_FOLDER=/mnt/dietpi_userdata/vaultwarden' /mnt/dietpi_userdata/vaultwarden/vaultwarden.env
 				# Create TLS certificate for web vault: Currently only RSA is supported. Add SAN with local IP and hostname, required for the client to accept the connection.
-				G_EXEC_OUTPUT=1 G_EXEC openssl req -reqexts SAN -subj '/CN=DietPi Bitwarden_RS' -config <(cat /etc/ssl/openssl.cnf <(echo -ne "[SAN]\nsubjectAltName=DNS:$(</etc/hostname),IP:$(mawk 'NR==4' /run/dietpi/.network)\nbasicConstraints=CA:TRUE,pathlen:0"))\
-					-x509 -days 7200 -sha256 -extensions SAN -out /mnt/dietpi_userdata/bitwarden_rs/cert.pem\
-					-newkey rsa:4096 -nodes -keyout /mnt/dietpi_userdata/bitwarden_rs/privkey.pem
-				G_CONFIG_INJECT 'ROCKET_TLS=' 'ROCKET_TLS={certs="./cert.pem",key="./privkey.pem"}' /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
-				G_CONFIG_INJECT 'ROCKET_PORT=' 'ROCKET_PORT=8001' /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env # Avoid port conflict with IceCast
-				G_CONFIG_INJECT 'WEB_VAULT_ENABLED=' 'WEB_VAULT_ENABLED=true' /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
+				G_EXEC_OUTPUT=1 G_EXEC openssl req -reqexts SAN -subj '/CN=DietPi Vaultwarden' -config <(cat /etc/ssl/openssl.cnf <(echo -ne "[SAN]\nsubjectAltName=DNS:$(</etc/hostname),IP:$(mawk 'NR==4' /run/dietpi/.network)\nbasicConstraints=CA:TRUE,pathlen:0"))\
+					-x509 -days 7200 -sha256 -extensions SAN -out /mnt/dietpi_userdata/vaultwarden/cert.pem\
+					-newkey rsa:4096 -nodes -keyout /mnt/dietpi_userdata/vaultwarden/privkey.pem
+				G_CONFIG_INJECT 'ROCKET_TLS=' 'ROCKET_TLS={certs="./cert.pem",key="./privkey.pem"}' /mnt/dietpi_userdata/vaultwarden/vaultwarden.env
+				G_CONFIG_INJECT 'ROCKET_PORT=' 'ROCKET_PORT=8001' /mnt/dietpi_userdata/vaultwarden/vaultwarden.env # Avoid port conflict with IceCast
+				G_CONFIG_INJECT 'WEB_VAULT_ENABLED=' 'WEB_VAULT_ENABLED=true' /mnt/dietpi_userdata/vaultwarden/vaultwarden.env
 			fi
 
 			# Permissions
-			G_EXEC chown -R bitwarden_rs:bitwarden_rs /mnt/dietpi_userdata/bitwarden_rs
-			G_EXEC chmod +x /opt/bitwarden_rs/target/release/bitwarden_rs
+			G_EXEC chown -R vaultwarden:vaultwarden /mnt/dietpi_userdata/vaultwarden
+			G_EXEC chmod +x /opt/vaultwarden/target/release/vaultwarden
 
-			# Service: https://github.com/dani-garcia/bitwarden_rs/wiki/Setup-as-a-systemd-service
-			cat << '_EOF_' > /etc/systemd/system/bitwarden_rs.service
+			# Service: https://github.com/dani-garcia/vaultwarden/wiki/Setup-as-a-systemd-service
+			cat << '_EOF_' > /etc/systemd/system/vaultwarden.service
```

Comment:
Old service needs to be removed.

Hmm, does it make sense to force a reinstall on next update, so that we don't need to have all the migration code in `dietpi-software`? Lately I try to avoid forced reinstalls, but here the migration is quite some more steps.

Reply:
it depends how we agree on this. At least this is a quite long running activity. But I agree. It might be better to force a controlled reinstall instead keeping such migration code for long

Reply:
ok service should be removed now if `/mnt/dietpi_userdata/bitwarden_rs` exist on reinstall